App/PyImports/Utils/Helpers.py

```diff
@@ -0,0 +1,38 @@
+def dict2xml(d, root_node=None):
+    wrap          =  False if None == root_node or isinstance(d, list) else True
+    root          = 'root' if None == root_node else root_node
+    root_singular = root[:-1] if 's' == root[-1] and None == root_node else root
```

Comment:
if `root_node == None`, your `root` is already set to `"root"`, so the first part of the conditional in line 4 is redundant!

Reply:
Also, please don't compare `None` with `==` - use `is` instead.
`if root_node is None` rather than `if root_node == None`
In general, use `is` when you want to check against an object's identity (e.g. checking to see if var is None). Use `==` when you want to check equality (e.g. Is var equal to 3?).
Your object you are comparing to `None` can have `__eq__` redefined thus rendering the conditional not obvious.

Reply:
Fixed. See dc4f8fced1eb47e179721c588f70737374027df3